Update metricflow-commands.md (#5290)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/metricflow-commands.md

- [`tutorial`](#tutorial) &mdash; Dedicated MetricFlow tutorial to help get you started.
-->

> <expandable alt_header="Check out this short video demo to run commands with the dbt Cloud CLI">

Author: ```suggestion
<expandable alt_header="How can I query or preview metrics with the dbt Cloud CLI?">
```
